tf-modules/archive/api.tf

```diff
@@ -82,6 +82,10 @@ resource "aws_api_gateway_rest_api" "api" {
   lifecycle {
     ignore_changes = [policy]
   }
+
+  endpoint_configuration {
```

Comment:
Would we need this for our distribution API as well? Or is that separate bc it's handled by TEA?

Reply:
Yep the API Gateway is deployed by TEA. I should check though how we are configuring TEA.

Reply:
Looks like we are good there